packages/calendar/stories/calendar.stories.ts

```diff
@@ -9,177 +9,71 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import { html, TemplateResult } from '@spectrum-web-components/base';
-import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import { html, render, TemplateResult } from '@spectrum-web-components/base';
+import { defaultLocale } from '@spectrum-web-components/story-decorator/src/StoryDecorator.js';
 
 import { spreadProps } from '../../../test/lit-helpers.js';
 
 import '@spectrum-web-components/calendar/sp-calendar.js';
-import '@spectrum-web-components/theme/sp-theme.js';
-
-const locales = [
-    'cs-CZ',
-    'cy-GB',
-    'da-DK',
-    'de-DE',
-    'en-GB',
-    'en-US',
-    'es-ES',
-    'fi-FI',
-    'fr-FR',
-    'hu-HU',
-    'it-IT',
-    'ja-JP',
-    'ko-KR',
-    'nb-NO',
-    'nl-NL',
-    'pl-PL',
-    'pt-BR',
-    'ru-RU',
-    'sv-SE',
-    'tr-TR',
-    'uk-UA',
-    'zh-Hans-CN',
-    'zh-Hans-CN-u-nu-hanidec',
-    'zh-Hant-TW',
-    'zz-ZY',
-    'zz-ZZ',
-] as const;
-
-const defaultLocale = 'en-US';
-
-const hiddenProperty = {
-    table: {
-        disable: true,
-    },
-};
 
 export default {
     title: 'Calendar',
     component: 'sp-calendar',
-
-    argTypes: {
-        lang: {
-            options: locales,
-            control: {
-                type: 'select',
-            },
-            table: {
-                defaultValue: {
-                    summary: defaultLocale,
-                },
-            },
-        },
-
-        // Don't render private properties and getters in the Storybook UI
-        currentDate: { ...hiddenProperty },
-        minDate: { ...hiddenProperty },
-        maxDate: { ...hiddenProperty },
-        weeksInCurrentMonth: { ...hiddenProperty },
-        weekdays: { ...hiddenProperty },
-        languageResolver: { ...hiddenProperty },
-        timeZone: { ...hiddenProperty },
-        locale: { ...hiddenProperty },
-        today: { ...hiddenProperty },
-
-        // Inherited
-        _dirParent: { ...hiddenProperty },
-        shadowRoot: { ...hiddenProperty },
-        dir: { ...hiddenProperty },
-        isLTR: { ...hiddenProperty },
-    },
-
-    args: {
-        lang: defaultLocale,
-    },
-
     parameters: {
-        controls: {
-            // Hide "This story is not configured to handle controls" warning
-            hideNoControlsWarning: true,
-        },
         actions: {
             handles: ['onChange'],
         },
     },
 };
 
-interface StoryArgs {
-    lang?: string;
-
-    padded?: boolean;
-    disabled?: boolean;
+type ComponentArgs = {
     selectedDate?: Date;
     min?: Date;
     max?: Date;
+    padded?: boolean;
+    disabled?: boolean;
+};
 
-    onChange?: (date: Date) => void;
+type StoryArgs = ComponentArgs & {
+    onChange?: (dateTime: Date) => void;
+};
 
+interface SpreadStoryArgs {
     [prop: string]: unknown;
 }
 
 const renderCalendar = (
     title: string,
```

Comment:
why title is not a part of args?

Reply:
I think what was meant here was to distinguish between the component attributes/properties and the <h1> used in each story. I will address this in a future PR.